pkg/image/api/install/install.go

```diff
@@ -95,7 +95,7 @@ func addVersionsToScheme(externalVersions ...unversioned.GroupVersion) {
 }
 
 func newRESTMapper(externalVersions []unversioned.GroupVersion) meta.RESTMapper {
-	rootScoped := sets.NewString("Image")
+	rootScoped := sets.NewString("Image", "ImageSignature")
```

Comment:
> Got any ideas for shortcut?

This is infrequent and most retrieval will usually be done through an istag.  I don't see a need.